src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateDirectionQuestionnaireLipCallBackHandler.java

```diff
@@ -47,6 +48,10 @@ public List<CaseEvent> handledEvents() {
 
     private CallbackResponse prepareDirectionsQuestionnaire(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        if (RespondentResponseTypeSpec.FULL_ADMISSION == caseData.getRespondent1ClaimResponseTypeForSpec()) {
```

Comment:
can you use existing method in case data isFullAdmitClaimSpec instead please?

Reply:
also would have done it differently.. would have had a flag on camunda so it is clear when the dq is generated rather than hidden in the handler. but i think ok because of the time constraints. in addition can you do if(!caseData.isFullAdmitClaimSpec()) { //generate form} and return AbourtToiStartOrSubmitCallbackResponse outside the condition? (because we are returning the same thing for both conditions)

Reply:
replaced the method